packages/node-core/src/indexer/storeCache/cacheModel.ts

```diff
@@ -46,6 +47,7 @@ export class CachedModel<
   private setCache: SetData<T> = {};
   private removeCache: Record<string, RemoveValue> = {};
   readonly hasAssociations: boolean = false;
+  private csvExporters: CsvStoreService[] = [];
```

Comment:
I think this can be generalised further. This could be for any sort of data sink, we could have ones for parquet or other dbs.

It would be good to have an interface rather than a type for this too